portfolio/src/main/webapp/style.css

```diff
@@ -3,23 +3,67 @@
     display: grid;
     grid-template-columns: 40% auto;
     grid-column-gap: 5%; 
+    padding-bottom: 3%;
+  }
+
+  #navbar, #footer {
+    grid-column-start: 1;
+    grid-column-end: span 2; 
+    padding-left: 10%;
+    padding-right: 40%; 
+    min-height: 5%; 
+  }
+
+  #navbar {
+    grid-row-start: 1; 
+  }
+
+  #footer {
+    grid-row-start: 3; 
   }
 
   #introduction {
+    grid-row-start: 2;
     grid-column-start: 1;
```

Comment:
Same here and below when it comes to sorting the lines

Reply:
Done.